flexmock.py

```diff
@@ -1118,6 +1138,8 @@ def _setattr(obj, name, value):
             local_override = True
         obj.__dict__[name] = value
     else:
+        if inspect.isclass(obj) and not vars(obj).get(name):
+            local_override = True
```

Comment:
Can you explain this one in a comment?

Reply:
I added a comment. What happens here is that we override a derived attribute on a subclass. This means that in teardown we just need to remove that attribute because it actually doesn't exist in that class.